cmd/merlinagent/main.go

```diff
@@ -67,9 +68,12 @@ const (
 
 func main() {
 
+	rand.Seed(time.Now().Unix())
+
 	flag.BoolVar(&verbose, "v", false, "Enable verbose output")
 	flag.BoolVar(&debug, "debug", false, "Enable debug output")
 	flag.StringVar(&url, "url", url, "Full URL for agent to connect to")
+	flag.Int64Var(&waitSkew, "skew", 3000, "varriable skew added to each agent sleep")
```

Comment:
Can we change the help message from `varriable skew added to each agent sleep` to `Variable time skew for agent to sleep`?

Reply:
Will work on those fixes now, thanks for being responsive!